melloddy_predictor/prepared_data.py

```diff
@@ -48,12 +49,14 @@ class PreparedData:
     _data: csr_matrix
     _df_failed: pd.DataFrame
     _compound_ids: pd.DataFrame
+    _num_cpu: int
 
     def __init__(
         self,
         encryption_key: pathlib.Path,
         preparation_parameters: pathlib.Path,
         smiles: pd.DataFrame,
+        num_cpu: int
```

Comment:
```suggestion
        num_cpu: int,
```

This should make the CI pass on your PR.